samples/hybrid-connections/java/README.md

```diff
@@ -0,0 +1,12 @@
+# Azure Hybrid Connections samples for Java
+
+There are two sample applications that demonstrate the usage of Hybrid Connections API for exchanging text over web socket mode as well as HTTP mode from Java. The following are the steps required to run the sample apps:
+
+1. Install JDK 1.8 or higher.
+2. Install Apache Maven.
+3. Clone or pull the latest library code [here](https://github.com/Azure/azure-relay-java).
+4. In cmd, go to the project root of azure-relay-java, then run "mvn clean package".
+5. Copy/paste over the "...-jar-with-depencies.jar" file under "/target" and place it under the "/lib" folder in the sample apps.
```

Comment:
>"...-jar-with-depencies.jar" [](start = 23, length = 28)

Can you provide a bit more of the example file filter, perhaps `azure-relay-*-jar-with-dependencies.jar`